3_ProtectionProfile/PP_Config-OS.adoc

```diff
@@ -0,0 +1,333 @@
+
+.Consistency Rationale for threats and OSPs
+[cols=".^1,.^1",options="header"]
+|===
+
+|PP-Module Threats/OSPs	
+|Consistency Rationale
+
+|T.Casual_Attack 
+.3+|The threat of zero-effort impostor attempt and presentation attack with related OSPs are specific subsets of the T.LIMITED_PHYSICAL_ACCESS (i.e. impersonate the user authentication mechanisms) threat in the <<PP_OS_V4.3>>.
+|OSP.Enrol
+|OSP.Verification_Error
+
+|OSP.Protection	
+|This OSP is specific subsets of the T.LIMITED_PHYSICAL_ACCESS (i.e. direct and possibly destructive access to its storage media (for biometric data)) threat in the <<PP_OS_V4.3>>.
+
+|===
+
+.Consistency Rationale for Assumptions
+[cols=".^1,.^1",options="header"]
+|===
+|PP-Module Assumptions	    
+|Consistency Rationale
+
+|A.Alternative	
+.3+|All assumptions levied on the operational environment of biometric system (i.e. The computer) are consistent with security requirements in the <<PP_OS_V4.3>>. 
+|A.Authentication
+|A.User
+
+|===
+
+=== Consistency of Objectives
+
+The objectives for the biometric system and its operational environment are consistent with the <<PP_OS_V4.3>> based on the following rationale:
+
+.Consistency Rationale for TOE Objectives
+[cols=".^1,.^1",options="header"]
+|===
+|PP-Module TOE Objectives	
+|Consistency Rationale
+
+|O.BIO_Verification	
+.3+|These TOE Objectives are specific subsets of the O.INTEGRITY objective in the <<PP_OS_V4.3>>. 
+|O.Enrol
+
+|O.Protection	
+|This TOE Objective is specific subset of the O.PROTECTED_STORAGE objective in the <<PP_OS_V4.3>>.
+
+|===
+
+.Consistency Rationale for Environmental Objectives
+[cols=".^1,.^1",options="header"]
+|===
+|PP-Module Environmental Objectives	
+|Consistency Rationale
+
+|OE.Alternative	
+.4+|All Environmental Objectives levied on the operational environment of biometric system (i.e. The computer) are consistent with security requirements in the <<PP_OS_V4.3>>. 
+|OE.Authentication
+|OE.Protection
+|OE.User
+
+|===
+
+=== Consistency of Requirements
+
+The Biometric System (i.e. TSF in the <<BIOPP-Module>>) is comprised of biometric capture sensors and firmware/software that provide functions described in the <<BIOPP-Module>> TOE design. The Biometric System may be integral to the computer or an external (self-contained) device that has been linked/paired with the computer. The Biometric System is invoked by the computer as defined in the <<PP_OS_V4.3>> when user’s biometric characteristics is presented to the sensor. The Biometric System creates and stores the template or compares the features with the stored template and returns the verification outcome to the computer.
+
+The <<BIOPP-Module>> assumes that the computer satisfies SFRs defined in the <<PP_OS_V4.3>> so that the Biometric System can work as specified in the <<BIOPP-Module>>. This section explains which SFRs in the <<PP_OS_V4.3>> are directly relevant to the Biometric System security functionality.
```

Comment:
Replace:
"The <<BIOPP-Module>> assumes that the computer satisfies SFRs defined in the <<PP_OS_V4.3>> so that the Biometric System can work as specified in the <<BIOPP-Module>>."

with

"The <<BIOPP-Module>> assumes that the TOE satisfies SFRs defined in the <<PP_OS_V4.3>> so that the Biometric System can work as specified in the <<BIOPP-Module>>.

Note that in the case where the TOE platform is "extended" with additional biometric hardware/firmware/software the GPOS SFRs might/should extend to the new "larger TOE".